apps/web/components/sections/project/supporter-updates.tsx

```
@@ -0,0 +1,46 @@
+
+const SupporterUpdates: React.FC<SupporterUpdatesProps> = ({ updates }) => {
+  return (
+    <section className="mt-6 bg-gray-50 rounded-lg p-6 shadow-md border border-gray-200 relative">
+      <div className="absolute top-4 right-6 text-sm text-purple-600 font-medium flex items-center">
+        <HiShieldCheck className="w-4 h-4 mr-1" />
+        Exclusive Content
+      </div>
+      <h3 className="text-lg font-bold mb-4">Supporter Updates</h3>
+      <div className="space-y-4">
+        {updates.map((update, index) => (
+          <UpdateCard key={index} {...update} />
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)